workflows/microbiome/nanopore-allele-based-pathogen-identification/README.md

```diff
@@ -0,0 +1,3 @@
+# Nanopore _Allele-based Pathogen Identification_
```

Comment:
Why Nanopore here?

Reply:
because we use minimap2 for the mapping step

Reply:
Please explain then in the description